spec/super_good/solidus_taxjar/backfill_transactions_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "spec_helper"
+
+RSpec.describe SuperGood::SolidusTaxjar::BackfillTransactions do
+  describe ".call" do
+    subject { described_class.call }
+
+    let!(:complete_order) { create :completed_order_with_totals }
+    let(:api_spy) { instance_spy(::SuperGood::SolidusTaxjar::Api) }
+
+    around do |example|
+      ::SuperGood::SolidusTaxjar.test_mode = true
```

Comment:
I'm not sure but is this necessary? Looks like we're stubbing out the api call anyways.